README.md

```diff
@@ -71,7 +71,9 @@ export default Component.extend(InViewportMixin, {
 To apply an `.active` class to your `Component` when it enters the viewport, you can simply bind the `active` class to the mixed in property `viewportEntered`, like so:
 
 ```js
-export default Ember.Component.extend(InViewportMixin, {
+import Component from '@ember/component';
+
```

Comment:
Nice!  Looks like I missed a few imports.  If you wanted to add the import for the mixin, it might be helpful for others copying these blocks of code.  Otherwise 👍 

```
import InViewportMixin from 'ember-in-viewport';
```

Reply:
Good callout! I'll add those in now. Thanks

Reply:
Added